src/createStore.js

```diff
@@ -148,6 +148,21 @@ export default function createStore(reducer, preloadedState, enhancer) {
    */
   function dispatch(action) {
     if (!isPlainObject(action)) {
+      if (action) {
+        if (action.__boundContext__) {
+          throw new Error(
+            'You have called dispatch with a boundAction. ' +
```

Comment:
Good catches. I've updated them to use "bound action creator" which seems to match the docs and another string in the unit tests. 
```
it('wraps the action creators with the dispatch function', () => {
```